### PR TITLE
Display absolute time in task list and assignment list

### PR DESCRIPTION
**Pull request by fwoelk**
## :loudspeaker: Type of change
- [x] New feature (non-breaking change which adds functionality)

## :scroll: Description
The relative display of time was not helpful in many cases (i.e. unable to identify the newer tasks of two tasks displaying "updated 2 days ago"). The absolute time of the last update is displayed now in the task pool and in the assignment list

## :link: Related Issues
* Fixes # (552)

## :ballot_box_with_check: Checklist:

- [x] I have performed a self-review of my own code
- [x] I have run the auto code formatting scripts 
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the `CHANGELOG.md`

### CODE REVIEW

File: client/src/components/AbsoluteDateTime.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import { Tooltip } from 'antd'
+import { useFormatter } from '../hooks/useFormatter'
+import { useServerMoment } from '../hooks/useServerTimeOffset'
+
+interface AbsoluteDateTimeProps extends HTMLProps<HTMLTimeElement> {
+  created: Date
+  updated: Date
+}
+
+const AbsoluteDateTime = (props: AbsoluteDateTimeProps) => {
```
Comment (HenningCash) [inline client/src/components/AbsoluteDateTime.tsx:12]: The component name is a bit misleading. Something like `EntityModificationDates` maybe?
Everything else looks good! Nice job :)

**Review by HenningCash — APPROVED** [review]

File: client/src/components/AbsoluteDateTime.tsx (hunk 1 above)
Comment (erikhofer) [reply client/src/components/AbsoluteDateTime.tsx:12]: I think so, too

Comment (fwoelk) [reply client/src/components/AbsoluteDateTime.tsx]: renamed to ModifucationTime

**Review by erikhofer — APPROVED** [review]

**Comment by HenningCash** [comment]
:tada:

**Pull request merged**